modular_chomp/code/modules/mob/living/carbon/human/species/station/traits/negative.dm

```diff
@@ -101,3 +101,20 @@
 	name = "Carbon Dioxide Breather"
 	desc = "You breathe carbon dioxide instead of oxygen, much like a plant. Oxygen is not poisonous to you."
 	var_changes = list("breath_type" = GAS_CO2, "exhale_type" = GAS_O2, "ideal_air_type" = /datum/gas_mixture/belly_air/carbon_dioxide_breather)
+
+/datum/trait/negative/boneless
+	name = "Boneless"
+	desc = "You have no bones! Though your limbs are also much, much easier to gib in exchange. (Seriously this can result in one shot deaths and similar)"
+	cost = -4
```

Comment:
honestly if we're going by 'extremely severe traits' it should be -8 just like bioincompatible major negative trait

Reply:
I personally think, start it here, don't over-tune it.

If after players use it and decide "Wow this really sucks and is totally not worth it" _then_ make the value a stronger reduction.